android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeViewModel.kt

```diff
@@ -117,8 +125,28 @@ class PracticeViewModel(
         _eventWaitDBInit.value = false
     }
 
+    fun getDotsString(): String? {

```

Comment:
`BrailleDots.toString` can be called instead

Reply:
No, it returns different result

Reply:
i need dots number for toast message (to be spoken aloud)

Reply:
Anyway, this functionality should not be done this way